verl/interactions/base.py

```diff
@@ -37,7 +37,7 @@ async def start_interaction(self, instance_id: Optional[str] = None, **kwargs) -
             return instance_id
 
     async def generate_response(
-        self, instance_id: str, messages: List[Dict[str, Any]], **kwargs
+        self, instance_id: str, current_turn_data: dict, messages: List[Dict[str, Any]], **kwargs
```

Comment:
please update docs/sglang_multiturn/interaction_system.rst for new argument current_turn_data. Is it an optional argument? There are some testcases that do not input this argument.